packages/components/src/FormField/FormFieldWrapper.tsx

```diff
@@ -157,19 +163,25 @@ export function FormFieldWrapperMain({
 }
 
 export function FormFieldLabel({
-  placeholder,
-  identifier,
+  children,
+  htmlFor,
   style,
+  external = false,
 }: {
-  readonly placeholder?: string;
-  readonly identifier?: string;
+  readonly children?: ReactNode;
+  readonly htmlFor?: string;
   readonly style?: React.CSSProperties;
+  readonly external?: boolean;
 }) {
-  if (!placeholder) return null;
+  if (!children) return null;
 
   return (
-    <label className={styles.label} htmlFor={identifier} style={style}>
-      {placeholder}
+    <label
+      className={external ? styles.externalLabel : styles.label}
+      htmlFor={htmlFor}
```

Comment:
Love how when you click on the label, it focuses the input. Cool! I made a storybook example with 3 inputs and each label click focuses the corresponding input